CHANGELOG.md

```diff
@@ -21,29 +21,39 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Breaking Changes
 
+## Program [0.2.3] [PR #8](https://github.com/jup-ag/jup-lock/pull/5)
+
+### Added
+
+- Add new instruction `cancelVestingEscrow`.
+- Add support for `token2022` extensions, including: `TransferFeeConfig`, `TokenMetadata`, `MetadataPointer`, `ConfidentialTransferMint`, `ConfidentialTransferFeeConfig`, `PermanentDelegate`, `TransferHook`, `MintCloseAuthority`, `DefaultAccountState`.
```

Comment:
Pls write more details:
1. We add many new endpoints v2 
2. Endpoint creating_vesting_plan is breaking change, we add a new field for cancel_mode in parameters
3. escrow states also adding many new fields

Reply:
Updated, however I think 2. point is not a breaking changes, for example old escrow is just not able to be cancelled as the migration result of the number is 0, which equal to `NeitherCreatorOrRecipient`.

That's why I put the change to `Added` section.